tests/nightly/test_large_array.py

```diff
@@ -1168,6 +1169,80 @@ def check_correctness(mxnet_op, numpy_op, atol=1e-3):
     check_binary_broadcast()
 
 
+def test_linalg():
+    def get_large_identity_mat():
```

Comment:
this is a helper function can we move it to test_utils.py like this: https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/test_utils.py#L381

Reply:
done